win/packaging/extra.wxs.in

```diff
@@ -486,7 +486,7 @@
       </Component>
 
       <!-- add reference so mysql client won't get uninstalled and we have a shortcut pointing to nowhere-->
-      <ComponentRef Id="C.bin.mysql.exe"/>
+      <ComponentRef Id="C.bin.mariadb.exe"/>
```

Comment:
nope

Reply:
This was from before the duplication solution.

Reply:
I'll revert.

Reply:
Reverted in bd5310ad45dfbcec57f696b4ada04083786f94ec

Reply:
Reverted in bd5310ad45dfbcec57f696b4ada04083786f94ec